.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,70 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
Maybe add a comment where this is coming from?

Reply:
It was autogenerated by GitHub.  I turned it on from the Repository settings